### ISSUE TITLE
Document that Eliot does no disk I/O in logging thread (except that Destinations do so actually this is a little silly)

### ISSUE THREAD

**Issue by itamarst** [itamarst/eliot#35]

**Comment by exarkun** [comment]
This isn't really the case - at least not currently - is it?  If you add a `FileDestination` to the global `Logger`'s D`estinations` then _whatever_ thread calls `Logger.write` will call `Destinations.send` will call `FileDestination.__call__` will call `file.write`.  As far as I can tell, if you want any special thread-related behavior at all you have to take extra steps to arrange for it.

Perhaps this ticket dates from a time when `ThreadedWriter` (or `ThreadedFileWriter`) was more central?

**Comment by itamarst** [comment]
Yeah... it's more that _outside_ of destinations there's no file I/O, but given what destinations do that's sort of silly.

Definitely want to have ThreadedWriter that isn't Twisted-specific. I wonder if new non-broken queue in Python 3.7 has been backported.
